pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Describe your changes
+Please include a summary of the change and which issue is fixed.
+
+## Issue ticket number and link
+
+#### Checklist before requesting a review
+#### General
+- [ ] I have performed a self-review of my code.
+- [ ] Code changes are considered complete for the given story (i.e., the comments from code review are addressed)
+- [ ] The commit title follows conventional commit standards.
+- [ ] I have put the PR link into the planner.
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes.
+- [ ] Code builds cleanly without any errors or warnings
+ 
+#### Code Quality
+- [ ] I have run linting and there were no errors
+- [ ] The changes have been reviewed for accessibility compliance.
```

Comment:
what does this mean?

Reply:
I was trying to say the changes to the code pass accessibility checks. I will reword this so it makes more sense!